### DISCUSSION TITLE
Remedy Pro 2.0.0

### DISCUSSION THREAD

**Discussion by Naozumi520** [Naozumi520/Remedy#18] [General]
Thanks for keep following Remedy. This release added some features and fixes:

- Add chat overlay
<img width="1792" alt="chat_overlay" src="https://user-images.githubusercontent.com/52615455/235064647-7c13924a-2d47-497a-ba7a-d88b6f717324.png">

```
You can now show the chat overlay by pressing [control + `]
```

- Replace login method with web login
<img width="1012" alt="login" src="https://user-images.githubusercontent.com/52615455/235065229-c32c7388-9aac-46bb-ac91-4be7a06abbfa.png">

- [Display icon in dock in preference or login prompt](https://github.com/Naozumi520/Remedy/commit/a6240a3d473f9587e40969a5385534bab67c648e)

- [Fix pin/unpin button greyedout issues](https://github.com/Naozumi520/Remedy/commit/a90883a4ed0cc96be3e52abb22bea4cc5a9ebe18)

- [Fix draggable issues](https://github.com/Naozumi520/Remedy/commit/614fd91c24b01f56de135a72da92a139bc1d511c)

- [Fix preferences no range limits](https://github.com/Naozumi520/Remedy/commit/511d3444a0d5c8e7bb0981e8cc1ff2f2d8c0dd3b)

- Disable ugly overflow

- Use the new overlay feature (Powered by electron officially)

- Fixed #12 

- Fixed #14 

<hr /><em>This discussion was created from the release <a href='https://github.com/Naozumi520/Remedy/releases/tag/RemedyPro2.0.0'>Remedy Pro 2.0.0</a>.</em>

**Comment by yankovichv** [comment]
@Naozumi520, great app. You saved my life on Maс 🙏

Tell me, can I see the microphone status of the participants in the conversation if I turn off the display of their nicknames?

I don't want to see nicknames, I know all the members of the room, there are more than enough avatars. But I want to see the Mute and Deafen status.

**Reply by Naozumi520** [reply]
Well... Currently not and you're right. People should be able to see the microphone status when nicknames are hidden. Fix will expected to be release in 1-2 days. 🙏

**Reply by yankovichv** [reply]
Wow 🥰 Something else 🤣

1. Thank you for the show/hide overlay menu item. I'd like the app to always be running, but only see the overlay when it's needed. Having a hotkey would be just fantastic 🤤

2. How can I support your development? 🥹

**Reply by Naozumi520** [reply]
Thank you so much for your kind words and support! 😊

I'm glad that you enjoy the app and finding it useful. I'll definitely consider adding a hotkey feature for toggling the overlay. It's a great idea to make the app even more convenient to use!

As for supporting the development, I'm welcome for any pull request. Any contribution is greatly appreciated and will help me continue improving the app! 
You can also help by spreading the word about the app or give me a star 🤣
Once again, thank you for your support! 🥰

**Reply by yankovichv** [reply]
Catch a star 💫

And I shared a link to this repository in our Discord community of 1500 people - https://gang.manganum.app/MDdz1ynr

Once again, accept the support from the entire team at https://manganum.app/ 🙏

**Reply by Naozumi520** [reply]
I'm excited to announce the release of version 2.0.2, which includes new features like overlay toggling, microphone status, and other important fixes! 😊

As of now, the overlay toggling shortcut is not customizable, but rest assured, I'm actively working on implementing a feature that will allow users to personalize it according to their preferences in the near future. Thank you!

**Reply by yankovichv** [reply]
👋 I tested the latest release a bit! Everything is amazing 🥰

But it seems there are a couple of bugs — https://gang.manganum.app/F6hbnpbf :
1. Hide names work only for me, but not for another channel member. 
2. Small empty artifact near my avatar.

**Reply by Naozumi520** [reply]
Sorry for not updating for a while. I managed to fix the issue of the empty artifact, but I'm still uncertain about what's causing the problem with the hide names feature not working for others. I've tested it multiple times, and it appears to be affecting all of my friends. 🥲

**Comment by HydraxSkarrag** [comment]
Hi, is there a way to disable the chat overlay as long it is not customizable? Or a way to change it to an other keybinding by editing a file or something? Thanks in advance!
